playbooks/roles/tower/defaults/main.yml

```diff
@@ -29,6 +29,11 @@ tower_credential_bundle_github_name: "Tower Default Github Credentials"
 tower_credential_bundle_github_desc: "Default Tower Credential Bundle for Github"
 tower_credential_bundle_github_type: 'scm'
 
+aws_credential_list:
+  - "blank"
+blank_access_key_id: ''
+blank_secret_access_key: ''
```

Comment:
Note sure why these blank variables are needed?

Reply:
just a default, if we run without passing in the private repo, it won't fail, and will create some dummy empty credentials.

Reply:
Ok, not seeing these being used anywhere though so was wondering why we need to set them

Reply:
Would only get created if we didn't pass in the group_vars in the private repo, I figured it might have been nice to demonstrate how this could be used in code form, will need to document it in a README/SOP.

Reply:
Sure sounds good. No such thing as too much documentation :)